js/pages/myToDoList.js

```diff
@@ -650,6 +654,12 @@ const setModuleAttributes = (data, modules, collections) => {
     modules['Enter SSN'].hasIcon = false;
     modules['Enter SSN'].noButton = false;
     modules['Enter SSN'].estimatedTime = 'Less than 5 minutes'
+    
+    modules['Covid-19'].header = 'COVID-19'
+    modules['Covid-19'].description = 'This survey asks questions about your history of COVID-19, including any vaccinations you may have received and details about times you may have gotten sick with COVID-19.';
+    modules['Covid-19'].hasIcon = false;
```

Comment:
not sure what `hasIcon` and `noButton` are used for... for the sake of being consistent with other standalone surveys, I would only set `header`, `description`, and `estimatedTime` similar to what we do below for Biospecimen Survey

Reply:
Solved